codebase/superdarn/src.bin/tk/tool/rtsnd.1.0/loginfo.h

```diff
@@ -0,0 +1,7 @@
+/* loginfo.h
+   =========
+   Author: R.J.Barnes
```

Comment:
Is this correct? We should also have the appropriate copyright and disclaimer here as well

Reply:
```suggestion
   Author: R.J.Barnes

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

I don't know what to put here for copyright or institution... let alone a year.